src/api/app/controllers/webui/webui_controller.rb

```diff
@@ -236,7 +236,7 @@ def require_package
     return if params[:package].blank?
 
     begin
-      @package = Package.get_by_project_and_name(@project.to_param, params[:package],
+      @package = Package.get_by_project_and_name(@project.to_param, params[:package_name] || params[:package],
```

Comment:
I feel this change is not right... I'm trying to come up with an alternative, because the form is settings the package parameters in the `:package` param, not the package name.

Reply:
Fixed